packages/medusa/src/instrumentation/index.ts

```diff
@@ -66,72 +66,72 @@ export function instrumentHttpLayer() {
    * Instrumenting the route handler to report traces to
    * OpenTelemetry
    */
-  ApiRoutesLoader.traceRoute = (handler) => {
-    return async (req, res) => {
-      if (shouldExcludeResource(req.originalUrl)) {
-        return await handler(req, res)
-      }
+  // ApiRoutesLoader.traceRoute = (handler) => {
```

Comment:
will this be removed?

Reply:
Nope. Have to add it back to the new implementation. That's the next thing I will do now